aries-site/src/pages/foundation/accessibility.mdx

```diff
@@ -29,7 +29,13 @@ baseline audit of the accessibility of your application.
 
 The following collection of guides from the [HPE Product Accessibility Program Office](https://hpe.sharepoint.com/sites/F5/CTO/Office/Accessibility/Pages/index.aspx) help designers and developers create accessible products which meet the needs of HPE’s clients.
 
-<GuideCards />
+<GuideSection />
+
+## Accessible Product Design Videos
```

Comment:
I think we'd like to add this to `foundation.js` in structures under the Accessibility page sections so it can be indexed in search